data/vivek839.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Vivek nath",
+  "type": "personal",
+  "bio": "The java code , OpenSource,remote work ,Linux, every one to learn code  because it develop you logic power",
+  "links": [
+  {
+  "name": "Follow me on twitter",
+  "url": "https://twitter.com/Viveknath",
+  "icon": "FaTwitter"
+},
+{
+  "name": "Follow me on Github",
+  "url": "https://github.com/vivek839",
+  "icon": "Github"
```

Comment:
You may want to replace this with **' FaGithub '**, because this is an invalid icon :+1: 
```Suggestion
  "icon": "FaGithub"
```

Reply:
This is a good suggestion, but not a blocker so I will merge your Pull Request. 

If you wish to make changes to your Profile then please raise a new Pull Request - you may wish to take a look at the Editing Guide in our Docs before doing so: https://linkfree.io/docs/how-to-guides/editing 

as well as the Available Icons page someone previously suggested: https://linkfree.io/icons